Tests/conf.json

```diff
@@ -3152,7 +3151,6 @@
         "_comment2": "~~~ UNSTABLE ~~~",
         "ServiceNow": "Instance goes to hibernate every few hours",
         "Tenable.sc": "unstable instance",
-        "ThreatConnect v2": "unstable instance",
```

Comment:
why are you removing from skipped?

Reply:
To be sure the test playbook passed in the build process too

Reply:
so don’t forget to skip the test after you merge this PR.